source/_integrations/meater.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Meater Smart Meat Thermometer
+description: Instructions on how to integrate the Meater Temperature Probe within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: '2021.2'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@Sotolotl'
+ha_domain: meater
+---
+
+The Meater Smart Meat Thermometer integration allows for communicating with the [Meater Temperature Probe](https://meater.com/) from Apption Labs.
+
+## Configuration
+
+To use the integration, your probe must be connected to the Meater Cloud service.
+
+To enable the integration, simply set it up via the UI.
```

Comment:
Please avoid the term `simply`, while it might sound simple to you, it might be hard for somebody else (and thus condescending).

Please add a description on how to do that.

Reply:
Done 👍